aws-datastore/src/test/java/com/amplifyframework/datastore/syncengine/SyncProcessorTest.java

```diff
@@ -69,13 +78,17 @@
 public final class SyncProcessorTest {
     private static final long OP_TIMEOUT_MS = TimeUnit.SECONDS.toMillis(2);
     private static final long BASE_SYNC_INTERVAL_MINUTES = TimeUnit.DAYS.toMinutes(1);
+    private static final List<String> SYSTEM_MODEL_NAMES =
+        Observable.fromIterable(SystemModelsProviderFactory.create().models())
+            .map(Class::getSimpleName).toList().blockingGet();
```

Comment:
I discuss a `ForEach `utility below that may make sense, if we have to pull this same trick more than a time or two.